test/NuGet.Core.FuncTests/Dotnet.Integration.Test/DotnetRestoreTests.cs

```diff
@@ -96,30 +96,187 @@ public void DotnetRestore_WithAuthorSignedPackage_Succeeds()
                 var workingDirectory = Path.Combine(pathContext.SolutionRoot, projectName);
                 var projectFile = Path.Combine(workingDirectory, $"{projectName}.csproj");
 
-                _msbuildFixture.CreateDotnetNewProject(pathContext.SolutionRoot, projectName, " classlib");
+                _msbuildFixture.CreateDotnetNewProject(pathContext.SolutionRoot, projectName, "classlib -f netstandard2.0");
+
+                using (var stream = File.Open(projectFile, FileMode.Open, FileAccess.ReadWrite))
+                {
+                    var xml = XDocument.Load(stream);
+
+                    var attributes = new Dictionary<string, string>() { { "Version", "1.0.0" } };
+
+                    ProjectFileUtils.AddItem(
+                        xml,
+                        "PackageReference",
+                        "TestPackage.AuthorSigned",
+                        string.Empty,
+                        new Dictionary<string, string>(),
+                        attributes);
+
+                    ProjectFileUtils.WriteXmlToFile(xml, stream);
+                }
+
+                _msbuildFixture.RestoreProject(workingDirectory, projectName, args: string.Empty);
+            }
+        }
+
+#if IS_SIGNING_SUPPORTED
+        [Fact]
+        public async Task WithUnSignedPackageAndSignatureValidationModeAsRequired_Fails()
```

Comment:
Could you change the naming, make it start with "DotnetRestore_"?

Reply:
I remember Damon gave a feedback in my old PR that adding class name is redundant for every test method. I will be happy to make that change to follow the convention just like other tests.

Reply:
added `DotnetRestore_` to the new tests and pushed those changes.

Reply:
The naming convention for unit tests is `<method being tested>_<scenario for test>_<expected result>`. But for integration tests, "method being tested" doesn't exactly make sense, or when all tests in a class are testing a single method. In other words, the naming convention for unit tests may not make sense for integration tests.

Reply:
I see..Thanks for correcting me :)